Simulation/Simulation.cs

```diff
@@ -191,9 +202,19 @@ public void CompileSimulation()
             SimulationMessaging.LoadAttributes(m_strSimulationID);
             SimulationMessaging.Method = this.Method;
             SimulationMessaging.AddMessage(new SimulationMessage("Begin compile simulation: " + DateTime.Now.ToString("HH:mm:ss")));
```

Comment:
This call to SimulationMessaging specifically, likely needs to be removed. (All .AddMessage calls probably need to be removed.

Reply:
As discussed, we are not removing it because it is not getting used in the new web application